.github/workflows/publish.yml

```diff
@@ -0,0 +1,59 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v.*
+
+jobs:
+  sbt:
+    name: sbt publish
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+      - name: Set up JDK 11
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11.0-9
```

Comment:
We don't support Java 8 or it is anyways safe to build release using 11?

Reply:
ow, good catch, looks like we don't specify a target in the build so we should definitely go with jdk8